Tests/GitHubLabels.tests.ps1

```diff
@@ -296,34 +296,66 @@ if ($script:accessTokenConfigured)
             }
         }
 
-        Context 'Removing labels from an issue' {
-            Remove-GitHubLabel -OwnerName $script:ownerName -RepositoryName $repositoryName -Name "discussion" -Issue $issue.number
-            Remove-GitHubLabel -OwnerName $script:ownerName -RepositoryName $repositoryName -Name "question" -Issue $issue.number
-            Remove-GitHubLabel -OwnerName $script:ownerName -RepositoryName $repositoryName -Name "bug" -Issue $issue.number
-            $labelIssues = Get-GitHubLabel -OwnerName $script:ownerName -RepositoryName $repositoryName -Issue $issue.number
+        $null = Remove-GitHubRepository -OwnerName $script:ownerName -RepositoryName $repositoryName
+    }
+    Describe 'Removing labels on an issue'{
+        $repositoryName = [Guid]::NewGuid().Guid
+        $null = New-GitHubRepository -RepositoryName $repositoryName
+
+        $issueName = [Guid]::NewGuid().Guid
+        $issue = New-GitHubIssue -OwnerName $script:ownerName -RepositoryName $repositoryName -Title $issueName
+
+        $labelsToAdd = @('pri:lowest', 'pri:low', 'pri:medium', 'pri:high', 'pri:highest', 'bug', 'duplicate',
+        'enhancement', 'up for grabs', 'question', 'discussion', 'wontfix', 'in progress', 'ready')
+        Add-GitHubIssueLabel -OwnerName $script:ownerName -RepositoryName $repositoryName -Issue $issue.number -LabelName $labelsToAdd
 
+        Context 'For removing individual issues'{
+            Remove-GitHubIssueLabel -OwnerName $script:ownerName -RepositoryName $repositoryName -Name "discussion" -Issue $issue.number
+            Remove-GitHubIssueLabel -OwnerName $script:ownerName -RepositoryName $repositoryName -Name "question" -Issue $issue.number
+            Remove-GitHubIssueLabel -OwnerName $script:ownerName -RepositoryName $repositoryName -Name "bug" -Issue $issue.number
+            $labelIssues = Get-GitHubLabel -OwnerName $script:ownerName -RepositoryName $repositoryName -Issue $issue.number
+    
             It 'Should have added the expected number of labels' {
```

Comment:
The names for these tests are very generic.

Reply:
Not sure what you mean by this, any suggestions for better test names?

Reply:
It just seems like if I run the tests from the console and I see a test label like 'Should have added the expected number of labels', I know nothing about the expected behavior. The test labels are as informative as simply saying It 'should do the right thing'.

Reply:
Makes sense, I made them a bit more descriptive